### PR TITLE
net-mail/*: EAPI=8 bumps & pkgcheck fixes

### PR DESCRIPTION
**Pull request by fkobi**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-mail/qmhandle/qmhandle-1.3.2-r1.ebuild [hunk 1]
```diff
@@ -10,13 +10,11 @@ SRC_URI="https://downloads.sourceforge.net/qmhandle/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~sparc ~x86"
-IUSE=""
 
 RDEPEND="virtual/qmail
 	dev-lang/perl
 	sys-process/psmisc
 "
-DEPEND=""
```
Comment (a17r) [inline net-mail/qmhandle/qmhandle-1.3.2-r1.ebuild:19]: does this really build without any dependencies?

File: net-mail/qmhandle/qmhandle-1.3.2.ebuild [hunk 2]
```diff
@@ -10,13 +10,11 @@ SRC_URI="https://downloads.sourceforge.net/qmhandle/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm ~hppa ~ia64 sparc x86"
-IUSE=""
 
 RDEPEND="virtual/qmail
 	dev-lang/perl
 	sys-process/psmisc
 "
-DEPEND=""
```
Comment (a17r) [inline net-mail/qmhandle/qmhandle-1.3.2.ebuild:19]: does this really build without any dependencies?

File: net-mail/vqadmin/vqadmin-2.3.7-r1.ebuild [hunk 3]
```diff
@@ -11,9 +11,9 @@ SRC_URI="http://www.inter7.com/vqadmin/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-WEBAPP_MANUAL_SLOT="yes"
 KEYWORDS="~amd64 ~sparc ~x86"
-IUSE=""
+
+WEBAPP_MANUAL_SLOT="yes"
```
Comment (a17r) [inline net-mail/vqadmin/vqadmin-2.3.7-r1.ebuild:16]: That looks like something we would normally put directly atop the `inherit` line, even if `webapp.eclass` completely lacks documentation in that regard.

Also, check (and fix) HOMEPAGE.

**Review by a17r — CHANGES_REQUESTED** [review]
If you are going to touch EAPI-7 ebuilds for minor stylistic changes, at least bump them to EAPI-8 at the same time (with all the necessary care doing so).

That also means you do the revbump necessary for an EAPI change and leave the existing stable ebuild alone.

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild [hunk 4]
```diff
@@ -12,9 +12,7 @@ HOMEPAGE="http://untroubled.org/qmail-notify/"
 SLOT="0"
 LICENSE="GPL-2"
 KEYWORDS="amd64 ~hppa sparc x86"
-IUSE=""
 
-DEPEND=""
 RDEPEND="${DEPEND}
```
Comment (arkamar) [inline net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild:16]:
```suggestion
RDEPEND="
```
This is also not needed.

**Review by arkamar — CHANGES_REQUESTED** [review]

**Comment by arkamar** [comment]
All 3 were merged, look at https://github.com/gentoo/gentoo/commit/7279d36da8233714ed49d361d774d11614e15bb7 and two previous commits https://github.com/gentoo/gentoo/commit/ec86de6055d104bfdcbf39f62145efc483aa0c45 and https://github.com/gentoo/gentoo/commit/439e9a3498c34a6988246ae0877ccb2c6b810384.

**Pull request closed without merging**